pandas/core/arrays/sparse/array.py

```diff
@@ -234,7 +234,7 @@ class SparseArray(PandasObject, ExtensionArray, ExtensionOpsMixin):
         3. ``data.dtype.fill_value`` if `fill_value` is None and `dtype`
            is not a ``SparseDtype`` and `data` is a ``SparseArray``.
 
-    kind : {'int', 'block'}, default 'int'
+    kind : {'integer', 'block'}, default 'integer'
```

Comment:
is this actually true?

Reply:
the default is 'integer'

https://github.com/pandas-dev/pandas/blob/fe1ea14910c9afec638114fce12308cdf26784dd/pandas/core/arrays/sparse/array.py#L277-L286

and is passed onto `make_sparse`

https://github.com/pandas-dev/pandas/blob/fe1ea14910c9afec638114fce12308cdf26784dd/pandas/core/arrays/sparse/array.py#L1513-L1528